vllm/model_executor/weight_utils.py

```diff
@@ -250,6 +262,20 @@ def hf_model_weights_iterator(
             with open(param_path, "rb") as f:
                 param = np.load(f)
             yield name, torch.from_numpy(param)
+    elif load_format == "tensorizer":
+        logger.warning("Deserializing HuggingFace models are not optimized for "
+                       "loading on vLLM, as tensorizer is forced to load to CPU. "
+                       "Consider deserializing a vLLM model instead for faster "
+                       "load times. See the `tensorize_vllm_model.py` example "
```

Comment:
reference the actual path to the script here, so `tensorize_vllm_model.py` -> `examples/tensorize_vllm_model.py`